src/Objects/_PVector.js

```diff
@@ -0,0 +1,283 @@
+module.exports = function(options, undef) {
+  'use strict';
+
+  const TAU = 2 * Math.PI,
+        //TAU = options.PConstants.TAU,
```

Comment:
So what is that _options_ parameter about?
Original implementation gets _**PConstants**_ from it!